paddle/fluid/framework/ir/onednn/self_attention_fuse_pass.cc

```diff
@@ -53,6 +54,14 @@ void SelfAttentionFusePass::ApplyImpl(ir::Graph* graph) const {
   LOG(WARNING) << "No-avx512 or MKL supported!";
   return;
 #endif
+
+#ifdef PADDLE_WITH_DNNL
```

Comment:
This define can be deleted since if the code can go here, meaning these definitions are all met:
```
#if !defined(__AVX512F__) || !defined(PADDLE_WITH_MKLML) || \
    !defined(PADDLE_WITH_DNNL)
```